lib/rollbar/sidekiq.rb

```diff
@@ -1,36 +1,35 @@
 # encoding: utf-8
 
-PARAM_BLACKLIST = ['backtrace', 'error_backtrace', 'error_message', 'error_class']
+module Rollbar
+  PARAM_BLACKLIST = ['backtrace', 'error_backtrace', 'error_message', 'error_class']
 
-if Sidekiq::VERSION < '3'
-  module Rollbar
-    class Sidekiq
-      def call(worker, msg, queue)
-        begin
-          yield
-        rescue Exception => e
-          params = msg.reject{ |k| PARAM_BLACKLIST.include?(k) }
-          scope = { :request => { :params => params } }
+  class SidekiqErrorHandler
+    def self.handle(scope, e, params)
+      params = msg.reject{ |k| PARAM_BLACKLIST.include?(k) }
+      scope = { :request => { :params => params } }
 
-          Rollbar.scope(scope).error(e)
-          raise
-        end
+      Rollbar.scope(scope).error(e)
+    end
+
+    def call(worker, msg, queue)
+      begin
+        yield
+      rescue Exception => e
+        SidekiqErrorHandler.handle(scope, e, params)
```

Comment:
`scope` or `params` are not defined here. As I've commented above, this call should pass `msg` and `e` to `.handle`.